client/a8c-for-agencies/sections/sites/features/jetpack/jetpack-sites-dataviews.tsx

```diff
@@ -41,8 +41,16 @@ export const JetpackSitesDataViews = ( {
 }: SitesDataViewsProps ) => {
 	const translate = useTranslate();
 
-	const { showOnlyFavorites } = useContext( SitesDashboardContext );
-	const totalSites = showOnlyFavorites ? data?.totalFavorites || 0 : data?.total || 0;
+	const { showOnlyFavorites, showOnlyDevelopmentSites } = useContext( SitesDashboardContext );
+	const totalSites = ( () => {
+		if ( showOnlyFavorites ) {
+			return data?.totalFavorites || 0;
+		}
+		if ( showOnlyDevelopmentSites ) {
+			return data?.totalDevelopmentSites || 0;
+		}
+		return data?.total || 0;
+	} )();
```

Comment:
This is the most readable `totalSites` calculation I came to so far, but feel free to suggest a better way if you can think of any.